### DISCUSSION TITLE
Add support for Google Web Snippet bot to Bot-Aware ISR Fallback

### DISCUSSION THREAD

**Discussion by david-buck** [vercel/next.js#31482] [Ideas]
### Describe the feature you'd like to request

Currently requests from the Google Web Snippet user agent:

```Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36 Google-PageRenderer Google (+https://developers.google.com/+/web/snippet/)```

aren't served using `fallback: blocking` as other bots are. This can break previews for Android Messaging and Google Chat.

### Describe the solution you'd like

Add the Google Web Snippet bot to the list of bots that triggers the blocking fallback.

### Describe alternatives you've considered

1. Delaying render of the page
2. Serving generic page metadata

**Comment by styfle** [comment]
Thanks, we'll get this fixed https://github.com/vercel/next.js/pull/31521
